python_modules/libraries/dagster-shell/dagster_shell/ops.py

```diff
@@ -91,14 +92,17 @@ def echo_graph():
     return output
 
 
+@deprecated(breaking_version="1.9", additional_warn_text="Use Dagster Pipes subprocess instead.")
```

Comment:
```suggestion
@deprecated(breaking_version="0.25", additional_warn_text="Use Dagster Pipes subprocess instead.")
```

Reply:
Should we specifically call out `PipesSubprocessClient` so people know what to look for?

Reply:
was the suggestion above a typo?

> Should we specifically call out PipesSubprocessClient so people know what to look for?

i was debating between calling out PipesSubprocessClient vs general concept, as the op user may not know how to use the client. but i guess we can call out the api to make the deprecation warning consistent with other ones (we always call out the exact alternative api)

Reply:
Not a typo - the Dagster package itself is versioned 1.x, but integration packages are versioned 0.x.

> i was debating between calling out PipesSubprocessClient vs general concept, as the op user may not know how to use the client. but i guess we can call out the api to make the deprecation warning consistent with other ones (we always call out the exact alternative api)

👍 👍